docs/guide/db-migrations.md

```diff
@@ -409,6 +599,18 @@ The migration command comes with a few command-line options that can be used to
   or a path [alias](concept-aliases.md). The template file is a PHP script in which you can use a predefined variable
   named `$className` to get the migration class name.
 
+* `generatorTemplateFile`: array (defaults to `[
+        'create' => '@yii/views/createMigration.php',
+        'drop' => '@yii/views/dropMigration.php',
+        'add' => '@yii/views/addMigration.php',
+        'remove' => '@yii/views/removeMigration.php',
+        'create_join' => '@yii/views/createJoinMigration.php'
+  ]`), specifies template files for generating migration code automatically. See [Creating Migrations with Generators](#creating-migrations-with-generators)
+  for more details.
+  
+* `fields`: array (defaults to `[]`), specifies the fields column use to creating migration automatically. The format that it use when declaring any applicable schema it is 
+`COLUMN_NAME:COLUMN_TYPE:COLUMN_DECORATOR`, for example `--fields=name:string(12):notNull`, it specify string column with 12 size and not null.
```

Comment:
`it specify string column with 12 size and not null.` -> `it specifies a string column of 12 chars with not null constraint`